.travis.yml

```diff
@@ -1,29 +1,26 @@
 sudo: required
-dist: xenial
+dist: bionic
 language: python
-python: '2.7'
+python:
+    - "2.7"
```

Comment:
python: 3 ?

Reply:
No. The purpose of this pull request is Openstack Train compatibility. There is a lot more work to do for Python 3 compatibility.

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/d116c2de-911d-4612-a8d0-bfe287466b28-0c2e4917-c514-46b4-81aa-9a122eb9bad8-v1.png?transparent=1&palette=1&width=246)